cmake/InstallArangoDBJSClient.cmake

```diff
@@ -31,11 +31,24 @@ if (USE_ENTERPRISE)
   )
 endif ()
 
-# For the node modules we need all files:
+# For the node modules we need almost all files:
 install(
   DIRECTORY ${ARANGODB_SOURCE_DIR}/js/node
   DESTINATION ${CMAKE_INSTALL_DATAROOTDIR_ARANGO}/${ARANGODB_JS_VERSION}
   REGEX "^.*/eslint"                                       EXCLUDE
   REGEX "^.*/.npmignore"                                   EXCLUDE
   REGEX "^.*/.bin"                                         EXCLUDE
+  REGEX "^.*/package.*json"                                EXCLUDE
```

Comment:
```suggestion
```

Reply:
@kvs85 @joerg84 : I think the line I just removed was too much.
I can't see if that fixes the problem because we will need a new Docker image to test it.

Reply:
Fix worked.